ssh_proxy_server/forwarders/base.py

```diff
@@ -11,11 +14,17 @@ class BaseForwarder(Module):
     def __init__(self, session):
         super().__init__()
         self.server_channel = session.ssh_client.transport.open_session()
+        # if session.agent:   # Experimental
```

Comment:
We could add a command line parameter to enable agent forwarding to the remote server